src/css/application.css

```diff
@@ -471,27 +471,23 @@ body {
   height: 40vh;
 }
 
-.console__entry-active,
+.console__entry,
 .console__input {
   margin-bottom: 0.5em;
 }
 
-.console__entry-active {
+.console__entry {
   margin-left: 4px;
 }
 
-.console__error-active {
+.console__error {
   color: var(--color-red);
 }
 
-.console__entry-inactive {
+.console__entry_inactive * {
```

Comment:
I am trying to gray the whole entry including the red error message. This seems like not the right way to do it. Should I pass a prop down to console output and be more specific?